ui/admin/app/templates/application.hbs

```diff
@@ -132,33 +132,43 @@
 {{/if}}
 
 <PendingConfirmations as |confirmation accept deny|>
-  <Rose::Dialog
-    @heading={{if
-      confirmation.options.title
-      (t confirmation.options.title)
-      (t 'actions.confirm')
-    }}
-    @dismissButtonText={{t 'actions.cancel'}}
-    @icon='flight-icons/svg/alert-triangle-16'
-    @style='warning'
-    @modal={{true}}
-    @dismiss={{deny}}
-    as |dialog|
-  >
-    <dialog.body>
-      <p>{{confirmation.text}}</p>
-    </dialog.body>
-    <dialog.footer>
-      <Rose::Button @style='primary' {{on 'click' accept}}>
-        {{if
-          confirmation.options.confirm
-          (t confirmation.options.confirm)
-          (t 'actions.ok')
-        }}
-      </Rose::Button>
-      <Rose::Button @style='secondary' {{on 'click' deny}}>
-        {{t 'actions.cancel'}}
-      </Rose::Button>
-    </dialog.footer>
-  </Rose::Dialog>
+  <Hds::Modal @color='warning' as |M|>
+    <M.Header
+      @dismissButtonText={{t 'actions.cancel'}}
+      @modal={{true}}
+      @dismiss={{deny}}
+      data-test-modal
+    >
+      {{if
+        confirmation.options.title
+        (t confirmation.options.title)
+        (t 'actions.confirm')
+      }}
+    </M.Header>
+    <M.Body>
+      <Hds::Text::Body @tag='p'>
+        {{confirmation.text}}
+      </Hds::Text::Body>
+    </M.Body>
+    <M.Footer>
+      <Hds::ButtonSet {{style width='15rem'}}>
```

Comment:
There is a reason why we are using inline styles here? I don't think its a pattern we use within the codebase.

Reply:
I based my refactor on the helios [docs](https://helios.hashicorp.design/components/button?tab=code#loading-state), but open to suggestions if the team has a preferred way
<img width="865" alt="Screenshot 2024-12-13 at 5 00 56 AM" src="https://github.com/user-attachments/assets/6939c0ba-2c81-4bad-bfa8-4be1b14d3e81" />

Reply:
Within our codebase, we have no inline styles, all our styles are defined within scss files. 
So unless there is an specific reason for it, or I will avoid them, or I will bring the topic to tech time to look for team consensous on introducing this approach.

Reply:
We shouldn't use an inline style. If you check out the note above in your doc link, HDS actually even says to not add an explicit width to the button (also this was added to the button set, not button) and called it out for a special use case when loading indicators can cause the button to change size.